wires/plugins/highlights/plugin.py

```diff
@@ -113,13 +113,17 @@ async def on_message(event: hikari.GuildMessageCreateEvent) -> None:
     guild = unwrap(event.get_guild())
     for user, triggers in notifications.items():
         channel = await plugin.app.rest.create_dm_channel(user)
+        footer = guild.name
+        if (source_channel := event.get_channel()):
+            if (source_channel_name := source_channel.name):
+                footer += f" | {source_channel_name}"
```

Comment:
```suggestion
            if source_channel.name:
                footer += f" | {source_channel.name}"
```

Reply:
Cleaned this up further by using `and` instead of nested `if` statements.